File: Data Structures & Algorithms/majority-element/submission-4.java
class Solution {
    public int majorityElement(int[] nums) {
        int sum = 0; 
        int n = nums.length / 2; 
        HashMap<Integer, Integer> mapping = new HashMap<>(); 
        for(int i = 0; i < nums.length; i++){
            mapping.put(nums[i], mapping.getOrDefault(nums[i], 0) + 1);
        }
        for(Integer key: mapping.keySet()){
            if(mapping.get(key) > n){
                return key;
            }
        } 
        return -1; 
    }

}

File: Data Structures & Algorithms/majority-element-ii/submission-4.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        counter = Counter(nums)
        calc = len(nums) / 3 
        res = [] 
        for key, value in counter.items():
            if value > calc:
                res.append(key)

        return res